lnbits/wallets/opennode.py

```diff
@@ -136,18 +133,20 @@ async def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
             yield value
 
     async def webhook_listener(self):
+        logger.error("webhook listener for opennode is disabled.")
+        return
         # TODO: request.form is undefined, was it something with Flask or quart?
         # probably issue introduced when refactoring?
-        data = await request.form  # type: ignore
-        if "status" not in data or data["status"] != "paid":
-            raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
-
-        charge_id = data["id"]
-        x = hmac.new(self.auth["Authorization"].encode("ascii"), digestmod="sha256")
-        x.update(charge_id.encode("ascii"))
-        if x.hexdigest() != data["hashed_order"]:
-            logger.error("invalid webhook, not from opennode")
-            raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
-
-        await self.queue.put(charge_id)
-        raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
+        # data = await request.form  # type: ignore
```

Comment:
- why disable code?
- will it later be re-enabled? if not, then it should be remove.

Reply:
same here